source/docs/hardware-reference/pigeon2/index.rst

```diff
@@ -31,25 +31,174 @@ Status Light Reference
-+---------------+------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+.. raw:: html
+
+    <style>
+        .led {
+            float: left;
+            height: 20px;
+            width: 20px;
+            border: 1px solid black;
+            border-radius: 10px;
+            margin: 5px;
+            background-color: darkgray;
+        }
+        .ledGroup {
+            display: inline-block;
+            height: 20px;
+        }
+        table.center, table.center th, table.center td {
+            border: 1px solid white;
+            border-collapse: collapse;
+            padding: 5px;
+            text-align: center;
+        }
+
+        .tableOverflow {
+            overflow: scroll;
+        }
+
+        td.overflow {
+            max-width: 550px;
+            overflow: scroll;
+        }
+
+        @media screen and (max-width: 480px) {
+            td.overflow {
+                max-width: 0;
+                overflow: scroll;
+            }
+
+            .tableOverflow {
+                max-width: 480px;
+            }
+        }
+    </style>
+
+    <div class="tableOverflow">
+        <table class="center">
+            <tr>
+                <th colspan="3">Blink Codes</th>
+            </tr>
+            <tr>
+                <th>Animation (Click to animate)</th>
+                <th>LED State</th>
+                <th>Meaning</th>
+            </tr>
+            <tr>
+                <td><div class='ledGroup'><div class='led' ontime='0' offtime='0' oncolor='black' offcolor='black'></div>
+                                          <div class='led' ontime='0' offtime='0' oncolor='black' offcolor='black'></div></div></td>
+                <td>LEDs Off</td>
+                <td>No Power</td>
+            </tr>
+            <tr>
+                <td><div class='ledGroup'><div class='led' ontime='300' offtime='300' oncolor='red' offcolor='black'></div>
+                                          <div class='led' ontime='300' offtime='300' oncolor='black' offcolor='red'></div></div></td>
+                <td>Blinking Alternating Red</td>
+                <td>Pigeon 2 does not have valid CAN.</td>
+            </tr>
+            <tr>
+                <td><div class='ledGroup'><div class='led' ontime='300' offtime='300' oncolor='orange' offcolor='black'></div>
+                                          <div class='led' ontime='300' offtime='300' oncolor='black' offcolor='orange'></div></div></td>
+                <td>Blinking Alternating Orange</td>
+                <td>Pigeon 2 detects CAN and does not see robot enable. Phoenix is not running in robot controller <b>or</b> Pigeon 2 does not have good CAN connection to robot controller.</td>
```

Comment:
> Pigeon 2 detects CAN and does not see robot enable. Phoenix is not running in robot controller <b>or</b> Pigeon 2 does not have good CAN connection to robot controller.

I think it's better if this is something along the lines of "Pigeon 2 detects CAN but cannot see Phoenix Diagnostics."

Then the "or" becomes broken into a "what causes it" block.

Reply:
The issue isn't necessarily Phoenix Diagnostics, it's specifically a robot program with Phoenix in general. The enable frame is what switches it over to simultaneous, not diagnostics.

Reply:
Customers have no idea what "Phoenix" is and I think the descriptions should somewhat reflect it. The description here is not clear to me.

Reply:
Given this is the Phoenix 6 documentation, I'd hope people know what Phoenix is at this point. I'm not sure saying "Phoenix 5 or 6" would be any clearer.